modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -22,6 +22,8 @@ import { encodePsbtMusig2ParticipantsKeyValData } from '../Musig2';
 export interface WalletUnspent<TNumber extends number | bigint = number> extends Unspent<TNumber> {
   chain: ChainCode;
   index: number;
+  witnessScript?: string;
```

Comment:
Are these just to let us put these properties on when we find them? How do we find witnessScript if the unspent returned from WP doesn't have it (does that ever happen?)?

Reply:
These are the properties that _may_ returned to us when we build a btc transaction, under txInfo. I left it optional since I am not too sure if every response to /build for **every** btc transaction returns unspent info with all these fields.

Reply:
Right, so my question is: are we sure they are present in all the cases we need them?

Reply:
We actually dont _need_ this field for our use case. We need txInfo.unspents to sign the unsigned commit txn hex and wanted to improve the type using the response from /build. However, during [signing](https://github.com/BitGo/BitGoJS/blob/214eafe48e8d12fd5d58efac289bab33bbd46fd3/modules/utxo-lib/src/bitgo/wallet/Unspent.ts#L37-L61) we care about the unspent index/chain/value/address.


```
 "unspents": [
      {
        "chain": 21,
        "index": 80,
        "**witnessScript**": "522102b2390579afa56499ebb1d5f92da6942c15e6acaae501a5d84c6cca433764d1bb21021fea984d5c20c716c14100bf4cd0e4bd0b851b540c22aeb2f010114eb90aeae121038299d563f8ee1aea4de65b1c23c352e3443ddb7f4355c39f989b0aaed7e2af8553ae",
        "id": "29182610ce831c9390b45bf0452a018ad39347c53289bf13cd996ee5ef2d4037:1",
        "address": "tb1q44kr6nhkgdxuu89n82sjpcskmltc7qgarzv0ftn7qgu3efw9n48s89egv0",
        "value": 1353750,
        "**valueString**": "1353750"
      }
    ],
```